cylc/flow/scripts/reinstall.py

```diff
@@ -220,6 +221,7 @@ def reinstall_cli(
         print(
             cparse('<magenta>Reinstall canceled, no changes made.</magenta>')
         )
+        return False
```

Comment:
This return is in the right place, it gets called if `use != 'n'` so catches the previous case.